adafruit_magtag/network.py

```diff
@@ -50,10 +55,10 @@ class Network(NetworkBase):
     def __init__(
         self,
         *,
-        status_neopixel=None,
-        extract_values=True,
-        debug=False,
-    ):
+        status_neopixel: Optional[str] = None,
```

Comment:
Should the be `Optional[neopixel.NeoPixel]`?

Reply:
I did some sleuthing in the REPL on my PyPortal, and found that `board.NEOPIXEL` is of type `Pin`. I'll paste the screenshot below. My interpretation of the function we're annotating is that it will take the `status_neopixel` argument either of two ways:
1. By passing it an initialized NeoPixel having type `neopixel.NeoPixel`
2. By passing it a Pin, like `board.NEOPIXEL` or `board.D10`, which the function then uses to create a `neopixel.NeoPixel`
So now I'm thinking it could be `Optional[Union[board.Pin, neopixel.NeoPixel]]`.
What are your thoughts on that?

<img width="640" alt="dir_board" src="https://user-images.githubusercontent.com/1261844/159142511-93080102-1cc1-4a61-aa73-dd409c80083f.png">

Reply:
I agree, only thing is that `Pin` is in the `microcontroller` module but otherwise yes that seems correct!

Reply:
Cool, thanks! I tracked down microcontroller.Pin when my previous push CI failed. Is there a method to tell what module a particular class is in?

Reply:
@UnicycleDumpTruck good question. I am not sure of an easy way to look that up. 
```
import microcontroller
dir(microcontroller)
```
does show the `Pin` class being in there, but unfortunately this is not as helpful if you don't already have an idea of where something is. 

Searching the docs might be the best way atm: https://docs.circuitpython.org/en/latest/search.html?q=Pin&check_keywords=yes&area=default

this search for Pin does show microcontroller.Pin as the top result. But I suspect there may be some scenarios when the top result is not necessarily correct for the situation, so probably needs some double checking, but it's a good place to start.